object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionVisitor.java

```diff
@@ -54,4 +60,31 @@ public Void visitMethodInvocation(MethodInvocationTree node, Void p) {
     }
     return super.visitMethodInvocation(node, p);
   }
+
+  @Override
```

Comment:
Can we add Javadoc for this method, like we had before for the `reportError()` override?

Reply:
Good point; I put it back.